### PR TITLE
[new release] regenerate (0.2)

### PR DESCRIPTION
**Pull request by Drup**
Regenerate is a tool to generate test-cases for regular expression engines

- Project page: <a href="https://github.com/regex-generate/regenerate">https://github.com/regex-generate/regenerate</a>
- Documentation: <a href="https://regex-generate.github.io/regenerate/doc/0.2/">https://regex-generate.github.io/regenerate/doc/0.2/</a>

##### CHANGES:

* Move to dune
* Use Iter instead of Sequence

### CODE REVIEW

File: packages/regenerate/regenerate.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+maintainer: "Drup <drupyog@zoho.com>"
+authors: "Drup <drupyog@zoho.com>"
+license: "ISC"
+homepage: "https://github.com/regex-generate/regenerate"
+bug-reports: "https://github.com/regex-generate/regenerate/issues"
+dev-repo: "git+https://github.com/regex-generate/regenerate.git"
+doc: "https://regex-generate.github.io/regenerate/doc/0.2/"
+
+depends: [
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/regenerate/regenerate.0.2/opam]:
```suggestion
  "dune" {>= "1.2"}
```
Dune should not be tagged as a build dependency. Moreover according to your `dune-project` file, this package requires at least dune 1.2

**Comment by Drup** [comment]
@kit-ty-kate Should be good now. :)

**Comment by kit-ty-kate** [comment]
Thanks

**Comment by Drup** [comment]
Thanks!

**Pull request merged**